lib/oauth/signature/base.rb

```diff
@@ -16,21 +16,6 @@ def self.implements(signature_method = nil)
       OAuth::Signature.available_methods[@implements] = self
     end
 
-    def self.digest_class(digest_class = nil)
-      return @digest_class if digest_class.nil?
-      @digest_class = digest_class
-    end
-
-    def self.digest_klass(digest_klass = nil)
-      return @digest_klass if digest_klass.nil?
-      @digest_klass = digest_klass
-    end
-
-    def self.hash_class(hash_class = nil)
```

Comment:
is there any doc on what this was replaced by?
We are still using 0.4.7 and I need to upgrade for ruby 3.2 ...